tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -50,24 +50,18 @@ sub run_test {
     upload_logs "$vnet_host_bridge_cfg_name";
     assert_script_run("rm -rf $vnet_host_bridge_cfg_name");
 
-    my $gi_host_bridge = '';
+    my ($mac, $nic);
     foreach my $guest (keys %xen::guests) {
         record_info "$guest", "HOST BRIDGE NETWORK for $guest";
         #figure out that used with virtio as the network device model during
         #attach-interface via virsh worked for all sles guest
-        assert_script_run("virsh attach-interface $guest network vnet_host_bridge --model virtio --live");
-        #Get the Guest IP Address from HOST BRIDGE NETWORK
-        if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
-            my $mac_host_bridge = script_output("virsh domiflist $guest | grep vnet_host_bridge|grep -oE \"[[:xdigit:]]{2}(:[[:xdigit:]]{2}){5}\"");
-            script_retry "ip neigh | grep $mac_host_bridge | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"", delay => 60, retry => 6, timeout => 60;
-            $gi_host_bridge = script_output("ip neigh | grep $mac_host_bridge | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"");
-        }
-        else {
-            script_retry "virsh domifaddr $guest --source arp | grep vnet0 | grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"", delay => 150, retry => 15, timeout => 150;
-            $gi_host_bridge = script_output("virsh domifaddr $guest --source arp | grep vnet0| grep -oE \"([0-9]{1,3}[\.]){3}[0-9]{1,3}\"");
-        }
+        $mac = '00:16:3e:32:' . (int(rand(89)) + 10) . ':' . (int(rand(89)) + 10);
+        assert_script_run("virsh attach-interface $guest network vnet_host_bridge --model virtio --mac $mac --live");
+        $nic = script_output "ssh root\@$guest ls /sys/class/net/ | grep -v 'lo\\|eth0'";
+        assert_script_run("ssh root\@$guest ip a a 192.168.122.209/24 dev $nic");
         #Confirm HOST BRIDGE NETWORK
-        assert_script_run("ssh root\@$gi_host_bridge 'ping -c2 -W1 openqa.suse.de'", 60);
+        assert_script_run("ping -c3 192.168.122.209");
```

Comment:
It would ping with the WLAN domain name (e.g openqa.suse.de) without the private IP (e.g 192.168.X.X) . 
 
Because of host bridge mode use an existing VM Host Server's network bridge - br0, then guest system will directly connect to the physical network.

Reply:
This line was a mistake. Can you please check the `test_network_interface()` function in `lib/virt_autotest/virtual_network_utils.pm` file now?